spec/requests/api/v2/providers/courses/update_accrediting_provider_spec.rb

```diff
@@ -0,0 +1,81 @@
+require "rails_helper"
+
+describe 'PATCH /providers/:provider_code/courses/:course_code' do
```

Comment:
This test for 
response `success`
and database has mad the change

but shouldn't it should also check that the response as the `accrediting_code ` changed as well

like it did in the original spec?

Reply:
perhaps, I'm not sure it matters much? We aren't using the returned value

Reply:
Yes; it shouldn't matter because we're not using it.